assets/js/monaco/styles.css

```diff
@@ -14,19 +14,10 @@
   src: url(codicon.ttf) format('truetype');
 }
 
-/* Fixes for curved borders (what a pain!)*/
+/* Background fix for transparent editor */
 .monaco-editor,
 .monaco-editor .overflow-guard {
   background-color: transparent !important;
-  @apply rounded-md;
-}
-.monaco-editor .margin {
-  @apply rounded-l-md;
-}
-.monaco-editor .monaco-scrollable-element {
-  @apply rounded-r-md;
-  border-top-right-radius: 0.375rem;
-  border-bottom-right-radius: 0.375rem;
```

Comment:
This looks like it would affect the LiveView one, and probably all the monaco instances, just want to make sure we know that and it's acceptable.